hw/ip/kmac/data/kmac.hjson

```diff
@@ -26,12 +26,12 @@
     { name: "fatal_fault_err",
       desc: '''
       This fatal alert is triggered when a fatal error is detected inside the KMAC unit.
-      Examples for such faults include
-      i) TL-UL bus integrity fault
-      ii) storage errors in the shadow registers
-      iii) errors in the message, round, or key counter
-      iv) any internal FSM entering an ivalid state
-      v) an error in the redundant lfsr
+      Examples for such faults include:
+      i) TL-UL bus integrity fault.
+      ii) Storage errors in the shadow registers.
+      iii) Errors in the message, round, or key counter.
+      iv) Any internal FSM entering an invalid state.
+      v) An error in the redundant lfsr.
```

Comment:
Hi Eunchan,
I added period here because of this generated diagram in the kmac doc. I feel it is easier to read if we have a period in the end.
![image](https://user-images.githubusercontent.com/11466553/151639095-b7e833df-314d-4759-a746-4d3f32b03273.png)